x-pack/plugins/apm/typings/TransactionGroup.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface ITransactionGroup {
```

Comment:
I can't figure out if this should live here, or in the `transaction_query` file.

Reply:
This is an interface that's shared between server and client so it seems to make sense to live here.

Reply:
Good point 👍